devito/dse/rewriters.py

```diff
@@ -406,6 +411,39 @@ def _extract_sum_of_products(self, cluster, template, **kwargs):
 
         return cluster.rebuild(processed)
 
+    @dse_pass
+    def _skewing(self, cluster, template, **kwargs):
+        """
+        Skew the accesses along the time Dimension.
+        """
+
+        skew_factor = self.DEFAULT_SKEW_FACTOR
+
+        _, sub_iterators, directions = cluster.ispace.args
+
+        skew_dim, mapper, intervals = None, {}, []
+
+        # Check if time dim exists
+        for i in cluster.ispace.intervals:
+            if i.dim.is_Time:
+                skew_dim = i.dim
+
+        if skew_dim is not None:
```

Comment:
if you use something along the lines of the code snippet above, you won't need this ``if`` anymore

Reply:
Got it. Thanks.